content/ja/news.md

```diff
@@ -14,21 +14,21 @@ _2021年1月30日_ -- [NumPy 1.20.0](https://numpy.org/doc/stable/release/1.20.0
 _2020年9月20日に_ 、私たちは[ NumPyプロジェクトにおけるダイバーシティやインクルージョンの状況や、ソーシャルメディア上での議論についての宣言 ](/diversity_sep2020)について書きました。
 
 
-### Natureに初めての公式のNumPy論文が掲載されました!
+### Natureに初の公式NumPy論文が掲載されました!
 
-_2020年9月16日_ -- NumPyに関する最初の公式の論文 [](https://www.nature.com/articles/s41586-020-2649-2) が査読付き論文として掲載されました。 これはNumPy 1.0のリリースから14年後のことになります。 この論文では、配列プログラミングのアプリケーションと基本的なコンセプト、NumPyの上に構築された様々な科学的Pythonエコシステム、そしてCuPy、Dask、JAXのような外部の配列およびテンソルライブラリとの相互運用を容易にするために最近追加された配列プロトコルについて説明しています。
+_2020年9月16日_ -- [NumPyに関する初の公式論文] (https://www.nature.com/articles/s41586-020-2649-2) が査読付き論文として掲載されました。 これはNumPy 1.0のリリースから14年後のことになります。 この論文では、配列プログラミングのアプリケーションと基本的なコンセプト、NumPyの上に構築された様々な科学的Pythonエコシステム、そしてCuPy、Dask、JAXのような外部の配列およびテンソルライブラリとの相互運用を容易にするために最近追加された配列プロトコルについて説明しています。
 
 
-### Python 3.9 が登場し、NumPy はいつバイナリホイールをリリースするのか?
+### Python 3.9 がリリースNumPyいつバイナリwheelをリリースするのか?
```

Comment:
I think previous translation is better.

Reply:
Maybe I fell asleep while editing this line 😅